ros2controlcli/ros2controlcli/verb/list_hardware_interfaces.py

```diff
@@ -25,17 +28,18 @@ def add_arguments(self, parser, cli_name):
         add_controller_mgr_parsers(parser)
 
     def main(self, *, args):
-        hardware_interfaces = list_hardware_interfaces(args.controller_manager)
-        command_interfaces = sorted(
-                hardware_interfaces.command_interfaces, key=lambda hwi: hwi.name)
-        state_interfaces = sorted(
-                hardware_interfaces.state_interfaces, key=lambda hwi: hwi.name)
-        print('command interfaces')
-        for command_interface in command_interfaces:
-            print('\t%s [%s]' %
-                  (command_interface.name,
-                   'claimed' if command_interface.is_claimed else 'unclaimed'))
-        print('state interfaces')
-        for state_interface in state_interfaces:
-            print('\t', state_interface.name)
-        return 0
+        with NodeStrategy(args) as node:
+            hardware_interfaces = list_hardware_interfaces(node, args.controller_manager)
+            command_interfaces = sorted(
+                    hardware_interfaces.command_interfaces, key=lambda hwi: hwi.name)
+            state_interfaces = sorted(
+                    hardware_interfaces.state_interfaces, key=lambda hwi: hwi.name)
+            print('command interfaces')
+            for command_interface in command_interfaces:
+                print('\t%s [%s]' %
```

Comment:
let's try always using format strings (aka  `f".... {bla}..."`) and if that doesn't fly (likely in this case) then use `... {}.format()`, it'd be good to get rid of as much hieroglyphs as possible (`%` in this case)

Reply:
the rest of the ros2cli tools are using the same style. We can make this a follow up issue, but I won't address it in this PR.